ci/unit/test_main_node.sh

```diff
@@ -73,5 +73,18 @@ fi
 
 rm -rf "$test_log_dir"
 
+# Use default python as that's where storage_estimator is installed.
+python3 -m venv venv
+# shellcheck disable=SC1091
+source venv/bin/activate
+# touch venv/pip.conf
+# pip config set global.progress_bar off
+# pip config set global.no_color true
```

Comment:
Is this meant to be left in?

Reply:
Yes, kind of.  The pip output is really messy in the logs but newer python as used by NLT accepts these options.. Python 3.6 however does not and storage_estimator is locked to 3.6 still even with this PR.  It's something I'd like to address but not in this PR - and at that point we'd uncomment these lines.